### ISSUE TITLE
Request parameters lost when calling via artisan command

### ISSUE THREAD

**Issue by Houshangian** [laravel/lumen-framework#1132]
- Lumen Version: 8.2.0
- PHP Version: 7.3

### Description:
when in artisan command create request with parameter to call a route, destination route dose not receive  request parameters.

### Steps To Reproduce:

this sample for test:

TestCommand:

> protected $signature = 'route:run';
public function handle()
    {
        $date=[
            'username' => 'test'
        ];
        $tokenRequest = Request::create('/test', 'post', $date);
        $res =app()->handle($tokenRequest)->withHeaders(['Content-Type'=>'application/json']);
        $responseBody = json_decode($res->getContent());
         var_dump($responseBody);
    }

TestController Method:

> public function test(Request $request)
    {
        $this->validate($request,[
           'username'=>'required'
        ]);
        return $request->all();
    }

And my route is:

> $router->post('/test','TestController@test');

when we call php artisan route:run in terminal, we have this message:

> object(stdClass)#474 (1) {
  ["username"]=>
  array(1) {
    [0]=>
    string(31) "The username field is required."
  }
}

**Comment by driesvints** [comment]
Hi there,

Thanks for reporting but it looks like this is a question which can be asked on a support channel. Please only use this issue tracker for reporting bugs with the library itself. If you have a question on how to use functionality provided by this repo you can try one of the following channels:

- [Laracasts Forums](https://laracasts.com/discuss)
- [Laravel.io Forums](https://laravel.io/forum)
- [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
- [Discord](https://discordapp.com/invite/KxwQuKb)
- [Larachat](https://larachat.co)
- [IRC](https://webchat.freenode.net/?nick=laravelnewbie&channels=%23laravel&prompt=1)

However, this issue will not be locked and everyone is still free to discuss solutions to your problem!

Thanks.

**Comment by Houshangian** [comment]
Tnx dear driesvints for your reply
But this issue is not a question, I just tried to figure out the situation which this bug happens.
we are using artisan command to call a route which is creating Request and populate it with some data, 
this fact easily happens when we are using laravel and in the destination we can get out parameters in the request  but it doesn't work perfectly in lumen and in destination we don't have any of our included parameters.
we search a lot on the we, there are some topics in stack overflow and ... that none of them include a response that tells us we are doing wrong. we also tried to ask experts that if we are doing it wrong but all of the guys we asked them, said that its not working perfect and its a bug. 
if you wish we can add the sample in github and send its link to you.
Really thank you in advance for your Attention

@driesvints 
@taylorotwell 
@JeffreyWay
